src/port1.0/portconfigure.tcl

```diff
@@ -527,9 +527,17 @@ proc portconfigure::get_compiler_fallback {} {
                 return {gcc-4.0}
             }
         } elseif {[vercmp $xcodeversion 3.0] >= 0} {
-            return {gcc-4.2 apple-gcc-4.2 gcc-4.0 macports-clang-3.4 macports-clang-3.3}
+            if {${configure.build_arch} eq "ppc" || ${configure.build_arch} eq "ppc64"} {
```

Comment:
Should this be testing if `${os.arch} eq "powerpc"` instead?

Reply:
I wondered the same thing for a bit -- but clang on an Intel machine can't make useful PPC code either, I figured. If you prefer the test to be os.arch I don't mind. Either will work on the PPC machines.

Reply:
I guess you're right, it should stay as it is.